README.md

```diff
@@ -238,6 +238,9 @@ test and produce json result files.
 There are some utilities for working with the json result files, run the `performance` script
 with no options to see them.
 
+## Run Accumulo on EC2 Test Cluster
```

Comment:
I think this title is misleading, running Accumulo on EC2 is what Muchos does.  The script automates setting up a snapshot version of Accumulo and Accumulo testing on a cluster.  I am not sure what a better title would be, that is also short.  Maybe `Automating cluster testing` would be a good title.

Reply:
OK, I just changed the front page readme to be consistent with your change.